SPECS-EXTENDED/kernel-rt/kernel-rt.spec

```diff
@@ -121,11 +121,12 @@ Requires:       audit
 %description tools
 This package contains the 'perf' performance analysis tools for Linux kernel.
 
-%package -n     python3-perf
+%package        python3-perf
```

Comment:
We should probably document it somewhere that we have specific `perf/bpftool` for `kernel-rt/hci`.